release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,2660 @@
+
+----
+# ansible-playbook -v -i <YOUR_INVENTORY>
+playbooks/byo/openshift-management/config.yml
+----
+
+[NOTE]
+====
+There is a link:https://bugzilla.redhat.com/show_bug.cgi?id=1506951[known issue] with this playbook.
+====
+
+To configure CFME 4.6 to consume the {product-title} installation it is running on:
+
+----
+# ansible-playbook -v -i <YOUR_INVENTORY>
+playbooks/byo/openshift-management/add_container_provider.yml
+----
+
+You can also automate the configuration of the provider to point to multiple OpenShift clusters:
+
+----
+# ansible-playbook -v -e container_providers_config=/tmp/cp.yml
+playbooks/byo/openshift-management/add_many_container_providers.yml
+----
+
+[NOTE]
+====
+The *_/tmp/cp.yml_* file requires some manual configurations to create and use
+it correctly. See
+xref:../install_config/cfme/container_provider.adoc#cfme-container-provider-multiple[Multiple
+Container Providers] for more information.
+====
+
+See xref:../install_config/cfme/index.adoc#install-config-cfme-intro[Deploying
+Red Hat CloudForms on OpenShift Container Platform] for more information.
+
+[[ocp-37-diagnostics]]
+=== Diagnostics
+
+[[ocp-37-additional-health-checks]]
+==== Additional Health Checks
+
+More health checks are now available for administrators to run after
+installations and upgrades. Administrators need the ability to run tests
+periodically to help determine the health of the framework components within the
+cluster. {product-title} 3.7 offers test functionality via Ansible playbooks
+that can be run and output can be sent as file-based output.
+
+----
+$ ansible-playbook playbooks/byo/openshift-checks/adhoc.yml
+                curator
+                diagnostics
+                disk_availability
+                docker_image_availability
+                docker_storage
+                elasticsearch
+                etcd_imagedata_size
+                etcd_traffic
+                etcd_volume
+                fluentd
+                fluentd_config
+                kibana
+                logging
+                logging_index_time
+                memory_availability
+                ovs_version
+                package_availability
+                package_update
+                package_version
+
+$ ansible-playbook playbooks/byo/openshift-checks/adhoc.yml -e
+openshift_checks=fluentd_config,logging_index_time,docker_storage
+----
+
+Alternatively, they are included in the health playbook:
+
+----
+$ ansible-playbook playbooks/byo/openshift-checks/health.yml
+----
+
+To capture the output:
+
+----
+$ ansible-playbook playbooks/byo/openshift-checks/health.yml -e
+openshift_checks_output_dir=/tmp/checks
+----
+
+[[ocp-37-metrics-and-logging]]
+=== Metrics and Logging
+
+[[ocp-37-journald-system-logs]]
+==== Jourald for System Logs and JSON File for Container Logs
+
+Docker log driver is set to `json-file` as the default for all nodes. Docker
+`log-driver` can be set to `journal`, but there is no log rate throttling with
```

Comment:
journal -> journald

Reply:
@juzhao Thanks, fixed in https://github.com/openshift/openshift-docs/pull/6556